src/PortfolioPlanning/Components/Plan/PlanPage.tsx

```diff
@@ -192,8 +192,10 @@ export default class PlanPage extends React.Component<IPlanPageProps, IPortfolio
     };
 
     private _backButtonClicked = (): void => {
-        this.props.toggleSelectedPlanId(undefined);
-        this.props.resetPlanState();
+        if (this.props.planLoadingStatus === LoadingStatus.Loaded) {

```

Comment:
I don't think the back button should be disabled. If for some reason the plan is taking too long to load or I clicked on the wrong plan I should be able to hit back whenever I want. Is the check for undefined planId in the saga not enough?

Reply:
The changes in the saga was enough already when I tested yesterday, I just left this disable back button here to be safe.
I think what you said make sense, I will delete this if check here.